bin/include/jvmdefaults.bat

```diff
@@ -42,40 +27,18 @@ if %java_version% GEQ 9 if %java_version% LSS 11 (
     %current_value%
 )
 
-if %java_version% GEQ 11 if %java_version% LSS 14 (
+if %java_version% GEQ 11 (
     set value= ^
     --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED ^
     --add-exports=java.base/sun.nio.ch=ALL-UNNAMED ^
     --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED ^
     --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED ^
     --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED ^
-    --add-opens=jdk.management/com.sun.management.internal=ALL-UNNAMED ^
-    --illegal-access=permit ^
-    %current_value%
-)
-
-if %java_version% GEQ 14 if %java_version% LSS 15 (
-    set value= ^
-    --add-exports=java.base/jdk.internal.misc=ALL-UNNAMED ^
-    --add-exports=java.base/sun.nio.ch=ALL-UNNAMED ^
-    --add-exports=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED ^
-    --add-exports=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED ^
-    --add-exports=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED ^
-    --add-opens=java.base/jdk.internal.access=ALL-UNNAMED ^
-    --add-opens=jdk.management/com.sun.management.internal=ALL-UNNAMED ^
-    --illegal-access=permit ^
-    %current_value%
-)
-
-if %java_version% GEQ 15 (
-    set value= ^
-    --add-opens=java.base/jdk.internal.access=ALL-UNNAMED ^
     --add-opens=java.base/jdk.internal.misc=ALL-UNNAMED ^
     --add-opens=java.base/sun.nio.ch=ALL-UNNAMED ^
     --add-opens=java.management/com.sun.jmx.mbeanserver=ALL-UNNAMED ^
     --add-opens=jdk.internal.jvmstat/sun.jvmstat.monitor=ALL-UNNAMED ^
     --add-opens=java.base/sun.reflect.generics.reflectiveObjects=ALL-UNNAMED ^
-    --add-opens=jdk.management/com.sun.management.internal=ALL-UNNAMED ^
```

Comment:
What is the reason for removing this option?

Reply:
This option was mistakenly omitted during refactoring, likely due to a misstep in managing the multi-line configuration. Will reinstate this option. Thanks for catching it.

Reply:
Hi @dy2285 ,

Also please check the 'jvmdefaults.sh` script.

Reply:
Hi @sk0x50 ,

Thanks for the follow-up. Checked the `jvmdefaults.sh` and there's no essential options were removed during the recent refactor. Please let me know if you have any additional concerns or if you'd like more details on the changes.

Reply:
Hi @dy2285 

Please take a look at https://github.com/gridgain/gridgain/pull/3241/files#r1715540549